src/components/Modal/ModalContent/MovieForm/MovieForm.jsx

```diff
@@ -1,78 +1,204 @@
+
+                console.log("err", err);
+            });
+        }
     };
 
     return(
-        <form action="#" className="movie-form">
-            <h1>
-                {props.isEditMovieForm ? (
-                    "Edit"
-                ): "Add"} movie
-            </h1>
-            <FormRow label="Title">
-                <input id="Title" placeholder="Title" type="text" onChange={inputTitle.onChange} value={inputTitle.value} />
-            </FormRow>
-            <FormRow label="Release date">
-                {/*TODO update date*/}
-                <input id="Release date" placeholder="Select date" type="date" {...(props.isEditMovieForm && { defaultValue: "10/12/2021" })} />
-            </FormRow>
-            <FormRow label="Movie URL">
-                <input id="Movie URL" placeholder="Movie URL here" type="text" onChange={inputMovieURL.onChange} value={inputMovieURL.value} />
-            </FormRow>
-            <FormRow label="Genre">
-                {/*TODO update select options*/}
-                <div className="select">
-                    <select id="genre">
-                        {selectOptions.map((option, index) => <option key={`option-${index}`} value={index}>{option}</option>)}
-                    </select>
-                </div>
-            </FormRow>
-            <FormRow label="Overview">
-                <input id="Overview" placeholder="Overview" type="text" onChange={inputOverview.onChange} value={inputOverview.value} />
-            </FormRow>
-            <FormRow label="Runtime">
-                <input id="Runtime" placeholder="Runtime" type="text" onChange={inputRuntime.onChange} value={inputRuntime.value} />
-            </FormRow>
-            <div className="buttons-holder">
-                <Button type="button" className="btn-outline" title="reset" onButtonClick={handleFormReset} />
-                {props.isEditMovieForm ? (
-                    <Button title="save" type="button"/>
-                ): <Button title="submit" type="submit"/> }
-            </div>
-        </form>
+        <Formik
+            initialValues={{
+                title: props.isEditMovieForm ? props.movie.title : "",
+                releaseDate: props.isEditMovieForm ? props.movie.releaseDate.toFormat("dd/MM/y") : "",
+                poster_path: props.isEditMovieForm ? props.movie.poster_path : "",
+                overview: props.isEditMovieForm ? props.movie.overview : "",
+                runtime: props.isEditMovieForm ? props.movie.runtime : "",
+                // genres: props.isEditMovieForm ? props.movie.genres : [],
+            }}
+            validateOnChange={false}
+            validate={values => {
+                const errors = {};
+
+                const date = DateTime.fromFormat(values.releaseDate, "dd/MM/y")
+
+                if (!date.isValid) {
+                    errors.releaseDate = "Date is invalid";
+                }
+
+                return errors;
```

Comment:
how about extracting this code to a separate function?

Reply:
done